gems/bootstrap-sass/CVE-2016-10735.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: bootstrap-sass
+cve: 2016-10735
+url: https://blog.getbootstrap.com/2018/12/13/bootstrap-3-4-0/
+title: XSS vulnerability in bootstrap-sass
+date: 2016-07-27
+
+description: |
+  In Bootstrap 3.x before 3.4.0 and 4.x-beta before 4.0.0-beta.2,
```

Comment:
Can you add an entry for `bootstrap` gem that covers this exact same issue but for the 4.x version(s)?

Reply:
Added as `gems/bootstrap/CVE-2016-10735.yml`